pkg/proxy/ipvs/proxier.go

```diff
@@ -1041,6 +1048,9 @@ func (proxier *Proxier) syncProxyRules() {
 				}
 
 				// ipvs call
+				if proxier.excludeExternalIP {
```

Comment:
This is technically loadbalancer IP and not external IP?

Reply:
It means both Service's ExternalIP and ingress IP